homeassistant/components/uptime/sensor.py

```diff
@@ -65,16 +58,4 @@ def state(self):
 
     async def async_update(self):
         """Update the state of the sensor."""
-        delta = dt_util.now() - self.initial
-        div_factor = 3600
-
-        if self.unit_of_measurement == "days":
-            div_factor *= 24
-        elif self.unit_of_measurement == "minutes":
-            div_factor /= 60
-        elif self.unit_of_measurement == "seconds":
-            div_factor /= 3600
-
-        delta = delta.total_seconds() / div_factor
-        self._state = round(delta, 2)
-        _LOGGER.debug("New value: %s", delta)
+        return self._state
```

Comment:
We don't need this method, as there is no update happening. The core is able to handle this method missing.

Additionally, adding this to the entity will let HA know this sensor should not be polled.

```py
    @property
    def should_poll(self) -> bool:
        """Disable polling for this entity."""
        return False
```